pom.xml

```diff
@@ -165,6 +164,11 @@
                 <artifactId>micrometer-registry-prometheus</artifactId>
                 <version>${micrometer.version}</version>
             </dependency>
+            <dependency>
+                <groupId>net.jodah</groupId>
+                <artifactId>failsafe</artifactId>
+                <version>2.2.0</version>
```

Comment:
Should we use a version property here like the others?

Reply:
I didn't put it as a property because no one else should be using that property since it is now in the dependency-management section. Also it avoids extra jumping around when trying to find out which version a dependency is at.

Reply:
Still. I think consistency is better and having all versions defined in one place makes it easier to see